corehq/apps/app_manager/tests/data/case_list_form/case-list-form-suite-multiple-references.xml

```diff
@@ -189,27 +189,9 @@
         <locale id="forms.m1f0"/>
       </text>
     </command>
-    <instance id="casedb" src="jr://instance/casedb"/>
-    <instance id="commcaresession" src="jr://instance/session"/>
     <session>
       <datum id="case_id_new_suite_test_0" function="uuid()"/>
     </session>
-    <stack>
-      <create if="count(instance('commcaresession')/session/data/return_to) = 1 and instance('commcaresession')/session/data/return_to = 'm0' and count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/case_id_new_suite_test_0]) &gt; 0">
-        <command value="'m0'"/>
-        <datum id="case_id" value="instance('commcaresession')/session/data/case_id_new_suite_test_0"/>
-      </create>
-      <create if="count(instance('commcaresession')/session/data/return_to) = 1 and instance('commcaresession')/session/data/return_to = 'm0' and count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/case_id_new_suite_test_0]) = 0">
-        <command value="'m0'"/>
-      </create>
-      <create if="count(instance('commcaresession')/session/data/return_to) = 1 and instance('commcaresession')/session/data/return_to = 'm2' and count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/case_id_new_suite_test_0]) &gt; 0">
-        <command value="'m2'"/>
-        <datum id="case_id" value="instance('commcaresession')/session/data/case_id_new_suite_test_0"/>
-      </create>
-      <create if="count(instance('commcaresession')/session/data/return_to) = 1 and instance('commcaresession')/session/data/return_to = 'm2' and count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/data/case_id_new_suite_test_0]) = 0">
-        <command value="'m2'"/>
-      </create>
-    </stack>
```

Comment:
I don't think this should change at all since there is no end of form navigation